src/protagonist/Orchestrator.Tests/Features/IIIF/IIIFAuth1BuilderTests.cs

```diff
@@ -1,254 +1,254 @@
-﻿using System;

```

Comment:
I think this file has different line endings - could just be the filename has changed though

Reply:
Yeah it looks like it could have but when I've changed it locally and add to git index it's always showing as a noop.